frontend/src/static/js/v_summary.js

```diff
@@ -166,11 +166,13 @@ window.vSummary = {
 
       return `http://github.com/${
         REPOS[user.repoId].location.organization}/${
-        REPOS[user.repoId].location.repoName}/commits/${
-        REPOS[user.repoId].branch}?`
-                + `author=${user.name}&`
-                + `since=${slice.date}'T'00:00:00+08:00&`
-                + `until=${untilDate}'T'23:59:59+08:00`;
+        REPOS[user.repoId].location.repoName}${
```

Comment:
Really weird indentation here.

Reply:
Can you make it consistent?

Reply:
I have updated the code due to an error where commits of the same date points to the same commit page link.